services/appflowy-collaborate/src/group/broadcast.rs

```diff
@@ -216,10 +216,13 @@ impl CollabBroadcast {
 
                   trace!("[realtime]: send {} => {}", message, cloned_user.user_device());
                   if let Err(err) = sink.send(message).await {
-                    error!("fail to broadcast message:{}", err);
+                    tracing::error!("fail to broadcast message:{}", err);
```

Comment:
it's not necessary to add `tracing::` here

Reply:
removed